test/clt-tests/core/test-unfreeze-defrosts.rec

```diff
@@ -62,3 +62,11 @@ sleep 5; mysql -h0 -P9306 -e "select count(*) from orders\G"
 ––– output –––
 *************************** 1. row ***************************
 count(*): 10000
+––– input –––
+mysql -h0 -P9306 -e "drop table if exists t; create table t; freeze t; flush ramchunk t;" >/dev/null 2>&1 &
+––– output –––
+[1] %{NUMBER}
+––– input –––
+sleep 3; searchd --stop | grep TERM
+––– output –––
+[#!/[A-Za-z]{3}\s+[A-Za-z]{3}\s+[0-9]{1,2}\s+[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]{3}\s+[0-9]{4}/!#] [%{NUMBER}] stop: successfully sent SIGTERM to pid %{NUMBER}
```

Comment:
As discussed, it's better to do `sleep 1; true` or smth, so you can be sure that `FREEZE` actually did something since otherwise your final test `stop: successfully sent SIGTERM to pid %{NUMBER}` can pass even if you don't do `drop table if exists t; create table t; freeze t; flush ramchunk t` at all